app/electron/main.js

```diff
@@ -136,6 +137,10 @@ function createWindow () {
 
 app.on('ready', createWindow);
 app.on('window-all-closed', function () {
+  if (!isDev) {
```

Comment:
Instead of relying on this property again, it's better to check the `serverProcess` itself (and initialize it to `null` when declaring it).

Reply:
done